lib/src/enum/swipe_direction.dart

```diff
@@ -2,6 +2,7 @@ part of '../swipable_stack.dart';
 
 /// The type of Action to use in [SwipableStack].
 enum SwipeDirection {
+  none,
```

Comment:
I think we should use `null` instead of `SwipeDirection.none` Because `SwipeDirection` is used to control action not only for displaying. (e.g. in [SwipeStackController#next](https://github.com/MaddinMade/flutter_swipable_stack/blob/390572d940dcbd3f6064a45f436121f748f74fae/lib/src/swipable_stack.dart#L629) )
 
Could you remove it `SwipeDirection.none` and use `null` on the other code that uses `SwipeDirection` please?